themes/bootstrap3/templates/myresearch/checkedout.phtml

```diff
@@ -111,7 +111,11 @@
           <?php endif; ?>
 
           <?php
-            $coverDetails = $this->record($resource)->getCoverDetails('checkedout', 'small', $this->recordLinker()->getUrl($resource));
+          if (!empty($resource->getExtraDetail('ils_details')['id'])) {
+              $coverDetails = $this->record($resource)->getCoverDetails('checkedout', 'small', $this->recordLinker()->getUrl($resource));
+          } else {
+              $coverDetails['html'] = "<img src='" . $this->imageLink('InnReachCover.png') . "' alt='book cover -" . $resource->getTitle() . "' />";
```

Comment:
Agreed. This just appears to be a generic "cover not available" image, so I'm not sure if it's needed at all. You could locally adjust the no-cover image if you want to use this version instead of the default one. Taking it out should simplify things a little bit, and if you really think InnReach needs its own cover image, we could create a more generic mechanism to support that in a separate PR after the basic functionality is merged.